src/docs/admin/references/properties/scheduler.properties

```diff
@@ -264,6 +264,18 @@ pa.scheduler.auth.policy.refreshperiod.seconds=30
 #pa.scheduler.dataspace.defaultuser.localpath=
 #pa.scheduler.dataspace.defaultuser.hostname=
 
+# Is userspace based on impersonation (true) or path (false)
```

Comment:
Is ?

Reply:
the scheduler.properties file is added automatically from github scheduling. Modifications needs to be done inside the scheduling project if needed.